### DISCUSSION TITLE
封面图片下载

### DISCUSSION THREAD

**Discussion by ZO1DB3RG** [yutto-dev/yutto#288] [Q&A]
为什么在下载的过程中是有jpg文件的，但是下载完成以后jpg文件就消失了

**Comment by ZO1DB3RG** [comment]
![image](https://github.com/user-attachments/assets/e604d7e9-8f67-4f69-b2f2-2c82ceab5d61)

**Reply by ZO1DB3RG** [reply]
就像这样消失了

**Reply by SigureMo** [reply]
图片是作为视频封面的，不作为独立资源存储
